src/app/[locale]/viewer/_components/transfer/TargetRespositories.tsx

```diff
@@ -0,0 +1,129 @@
+                            <Typography variant="h4"
+                                        sx={{ minWidth: '200px', mr: 2 }}><FormattedMessage {...messages.mnestix.transfer.chooseRepository} /></Typography>
+                            <FormControl fullWidth>
+                                <Controller
+                                    name="repository"
+                                    control={control}
+                                    defaultValue=""
+                                    render={({ field }) => (
+                                        <TextField fullWidth 
+                                                   select
+                                                   label={intl.formatMessage(messages.mnestix.transfer.repositoryLabel)}
+                                                   required {...field}>{ aasRepositories.map((repo, index) => {
+                                            return <MenuItem key={index} value={repo}>{repo}</MenuItem>
+                                        })}
+                                        </TextField>)}
+                                />
+                            </FormControl>
+                        </Box>
+                        <Box display="flex" flexDirection="row" mt={2} alignItems="center">
+                            <Typography variant="h4"
+                                        sx={{ minWidth: '200px', mr: 2 }}><FormattedMessage {...messages.mnestix.transfer.chooseSubmodelRepository} /></Typography>
+                            <FormControl fullWidth>
+                                <Controller
+                                    name="submodelRepository"
+                                    control={control}
+                                    defaultValue="0"
+                                    render={({ field }) => (
+                                        <TextField fullWidth 
+                                                   select
+                                                   label={intl.formatMessage(messages.mnestix.transfer.submodelRepositoryLabel)}
+                                                   {...field}>
+                                            <MenuItem key="none" value="0"><FormattedMessage {...messages.mnestix.transfer.useAasRepository} /></MenuItem>
+                                            { submodelRepositories.map((repo, index) => {
+                                                return <MenuItem key={index} value={repo}>{repo}</MenuItem>
+                                            })}
+                                        </TextField>)}
+                                />
+                            </FormControl>
+                        </Box>
+                        <Divider sx={{ mt: 6, mb: 4 }}/>
+                        <DialogActions>
+                            <Button sx={{ mr: 1 }} variant="outlined" onClick={handleSubmit(onSubmit)}>Save & Back to Previous Aas</Button>
```

Comment:
The message is added to the i18n file but hardcoded here

Reply:
Maybe more a question for the UX review: Should this button be clickable when no Repo is selected? The default selection for the AAS repository is currently empty

Reply:
Was also a feedback from claudette and is now implemented like that